src/components/icon/js/iconDirective.js

```diff
@@ -210,11 +210,12 @@ function mdIconDirective($mdIcon, $mdTheming, $mdAria ) {
       // Use either pre-configured SVG or URL source, respectively.
       attr.$observe(attrName, function(attrVal) {
 
-        element.empty();
```

Comment:
Should we not keep this here also? To force clear all content WHILE loading the SVG?